packages/app/src/runner/useAutHeader.ts

```diff
@@ -0,0 +1,19 @@
+import { useElementSize } from '@vueuse/core'
+import { ref, watch } from 'vue'
+import { useAutStore } from '../store'
+
+export function useAutHeader () {
+  const autStore = useAutStore()
+  const autHeaderEl = ref<HTMLDivElement>()
+  const { height } = useElementSize(autHeaderEl)
+
+  watch(height, () => {
+    autStore.setSpecRunnerHeaderHeight(height.value)
+  }, {
+    immediate: true,
+  })
+
+  return {
+    autHeaderEl,
+  }
```

Comment:
We should consider building a full-on AutHeader component at some point (not this PR). There's enough shared logic now that is would clean up the header containers quite a bit, even if we have to disable most of the functionality in run mode.

Reply:
Agreed